attributes/geo:proj/README.md

```diff
@@ -0,0 +1,149 @@
+# Projection Attribute Extension for Zarr
+
+- **Extension Name**: Projection Attribute Extension
+- **Version**: 1.0.0
+- **Extension Type**: Attribute
+- **Status**: Proposed
+- **Owners**: @emmanuelmathot
+
+## Description
+
+This extension defines a standardized way to encode coordinate reference system (CRS) information for geospatial Zarr arrays and groups using the `geo:proj` attribute.
+
+## Motivation
+
+- Provides simple, standardized CRS encoding without complex nested structures
+- Addresses issues identified in GeoZarr discussions regarding CF convention complexity
+- Compatible with existing geospatial tools (GDAL, rasterio, pyproj)
+- Based on the proven STAC Projection Extension model
+
+## Specification
+
+The `geo:proj` attribute can be added to Zarr arrays or groups to define projection information.
+
+### Required Fields
+
+At least one of the following MUST be provided:
+
+- `code`: Authority and code identifier (e.g., "EPSG:4326")
+- `wkt2`: WKT2 string representation of the CRS
+- `projjson`: PROJJSON object representation of the CRS
+
+### Optional Fields
+
+- `bbox`: Bounding box in the CRS coordinates
+- `transform`: Affine transformation coefficients (6 or 9 elements)
+- `spatial_dims`: Names of spatial dimensions in the array
+
+Note: The shape of spatial dimensions is obtained directly from the Zarr array metadata once the spatial dimensions are identified.
+
+### Spatial Dimension Identification
+
+The extension identifies spatial dimensions through:
```

Comment:
It might be worth clarifying what "spatial dimensions" mean, i.e., that it corresponds to array dimensions and not spatial axes.

This may disambiguate cases like irregular data with `x(node)` and `y(node)` 1-dimensional explicit coordinates and `spatial_dims: [“node”]`

Reply:
Yes that's clearer!

Perhaps remove the `(axes)` in that paragraph? So the distinction between array dimension names (arbitrary names) vs. spatial axis names (X, Y) is even clearer.

Another relevant example taken from CF conventions 1.12 (example 5.2):

```
  float xc(xc) ;
    xc:axis = "X" ;
    xc:long_name = "x-coordinate in Cartesian system" ;
    xc:units = "m" ;
  float yc(yc) ;
    yc:axis = "Y" ;
    yc:long_name = "y-coordinate in Cartesian system" ;
    yc:units = "m" ;
```